analyzer/consensus/consensus.go

```diff
@@ -438,13 +438,12 @@ func (m *Main) queueEventInserts(batch *storage.QueryBatch, data *storage.Consen
 
 	for i := 0; i < len(data.Results); i++ {
 		for j := 0; j < len(data.Results[i].Events); j++ {
-			backend, ty, body, err := extractEventData(data.Results[i].Events[j])
+			_, ty, body, err := extractEventData(data.Results[i].Events[j])
```

Comment:
I did not change `extractEventData()` because it will be refactored in https://github.com/oasisprotocol/oasis-indexer/pull/188/files (heads up @aefhm) and I wanted to minimize merge conflicts.